subdomains/blog/_posts/2020-11-25-command-spotlight-volta-run.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Command Spotlight: volta run"
+date:   2020-11-23 08:00:00 -0700
+categories: [spotlight, commands]
+author: Chuck Pierce
+excerpt_separator: <!--more-->
+---
+
+Volta's context detection is designed to be invisible: You type a command—like `npm start`—and Volta automatically detects and runs the correct version of that command. Sometimes, however, you need to take direct control over the tool versions that are used. Maybe you are trying to reproduce reported bug or testing how your project works with different tool versions. Whatever the reason, **`volta run` lets you run any command while specifying the Node, npm, and Yarn versions right at the command line**
```

Comment:
- "Volta's context detection is designed to be invisible" => "Volta is designed to be invisible"
- "You type a command—like `npm start`—and Volta automatically detects and runs the correct version of that command." => "Type a command like `npm start`, and Volta automatically detects and runs the right version of `npm`."
- Play with a paragraph break before "Sometimes, however" and see if it feels like it flows a little more briskly.
- The bolded takeaway is too much detail. See if you can golf it down or abstract out the problem statement rather than the documentation details. Maybe something like "**`volta run` lets you take control for the cases where you need to be explicit about your tool selection.**"?

Reply:
Did some rewording here, let me know if it still doesn't quite flow well enough.